core/src/main/java/org/apache/iceberg/avro/ValueReaders.java

```diff
@@ -748,4 +753,17 @@ public Long read(Decoder ignored, Object reuse) throws IOException {
       return currentPosition;
     }
   }
+
+  static class ConstantReader<C> implements ValueReader<C> {
```

Comment:
@chenjunjiedada, since this is no longer used, can you please remove it?

Reply:
Sure，I should remove this first to track all its usage. I believe now no one is using it.